react/assets/components/SearchIcon.js

```diff
@@ -1,10 +1,17 @@
 import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
 import styles from './SearchIcon.css'
 class SearchIcon extends PureComponent {
   render() {
     return (
       <svg
-        className={`${styles.modalSearchIcon} pkpmodal-search-icon`}
+        className={
+          this.props.isGeolocation
+            ? `${styles.modalSearchIcon} pkpmodal-search-icon`
```

Comment:
⛏ Maybe you could put `pkpmodal-search-icon` outside the ternary operator, since it appears in both cases